doc/conf.py

```diff
@@ -0,0 +1,16 @@
+# Configuration file for the Sphinx documentation builder.
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+
+project = "imSim"
+copyright = "2016-2022, LSSTDESC"
+author = "LSSTDESC"
+
+
+extensions = ["sphinx.ext.napoleon", "sphinx_rtd_theme", "sphinx.ext.mathjax"]
```

Comment:
I'd recommend adding `sphinx.ext.autodoc` and `spinx.ext.viewcode`.  

I have a lot of other things in my TreeCorr conf.py (and similar in GalSim) that we might want to copy over.  E.g. I think we want to grab the version number from `imsim.__version__` or `imsim.__version_info__`.  
And I set `default_role = 'any'`, which is pretty convenient to get cross references working without too much overhead. 
I also have some extensions to the Napoleon style that I cribbed from [here](https://michaelgoerz.net/notes/extending-sphinx-napoleon-docstring-sections.html) which I like.  

cf. https://github.com/rmjarvis/TreeCorr/blob/releases/4.3/docs/conf.py